### ISSUE TITLE
piped makepkg conflicts with check() function

### ISSUE THREAD

**Issue by rmarquis** [rmarquis/pacaur#197]
It involves some kind of conflicts between that specific VCS package building with the piped makepkg and the check() routine. Building the package with "yes y | makepkg --nocheck" fixes this, but this is not an optimal workaround.

```
$ pacaur -Qua
:: aur  python-pywer-git  0.10.1.r5.gc76eb38-1  ->  0.11.r0.g8506dbb-1  
:: aur  python3-aur       2013.8-2              ->  2013.10-1
```

```
$ pacaur -Sua
:: Starting AUR upgrade...
:: resolving dependencies...
:: looking for inter-conflicts...
:: python-pywer-git latest revision -- fetching

AUR Packages  (2):

Name                  Old Version           New Version         

aur/python-pywer-git  0.10.1.r5.gc76eb38-1  latest                             
aur/python3-aur       2013.8-2              2013.10-1                          

:: Proceed with installation? [Y/n] y
:: View python3-aur PKGBUILD? [Y/n] n
:: View python-pywer-git PKGBUILD? [Y/n] n
:: Building python-pywer-git package...
==> Making package: python-pywer-git 0.11.r0.g8506dbb-1 (Wed Oct  2 11:46:25 CEST 2013)
==> Retrieving sources...
  -> Cloning pywer git repo...
Cloning into bare repository '/tmp/pacaurtmp-remy/python-pywer-git/pywer'...
remote: Counting objects: 654, done.
remote: Compressing objects: 100% (650/650), done.
remote: Total 654 (delta 431), reused 0 (delta 0)
Receiving objects: 100% (654/654), 114.35 KiB | 72.00 KiB/s, done.
Resolving deltas: 100% (431/431), done.
Checking connectivity... done
==> Validating source files with md5sums...
    pywer ... Skipped
==> Extracting sources...
  -> Creating working copy of pywer git repo...
Cloning into 'pywer'...
done.
==> Starting pkgver()...
==> Starting prepare()...
==> Sources are ready.
==> Making package: python-pywer-git 0.11.r0.g8506dbb-1 (Wed Oct  2 11:46:28 CEST 2013)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing src/ tree
==> Starting check()...
test_string_input_dlpkgs (test.pywertest.PrinterBadInput)
download_pkg should fail with string input as first arg ... ok
test_string_input_ppsi (test.pywertest.PrinterBadInput)
pretty_print_simple_info should fail with string input as first arg ... stty: standard input: Inappropriate ioctl for device
ERROR
test_string_input_ppu (test.pywertest.PrinterBadInput)
pretty_print_updpkgs should fail with string input pkgs arg ... ok
test_known_values (test.pkgbuildtest.PkgbuildTest)
parse_pkgbuild should return the values listed above ... ok
test_variable_expansion (test.pkgbuildtest.PkgbuildTest)
parse_pkgbuild should return the values with variables substituted ... ok

======================================================================
ERROR: test_string_input_ppsi (test.pywertest.PrinterBadInput)
pretty_print_simple_info should fail with string input as first arg
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/pacaurtmp-remy/python-pywer-git/src/pywer/test/pywertest.py", line 18, in test_string_input_ppsi
    self.assertRaises(TypeError, printer.pretty_print_info, 'foo')
  File "/usr/lib/python3.3/unittest/case.py", line 570, in assertRaises
    return context.handle('assertRaises', callableObj, args, kwargs)
  File "/usr/lib/python3.3/unittest/case.py", line 135, in handle
    callable_obj(*args, **kwargs)
  File "/tmp/pacaurtmp-remy/python-pywer-git/src/pywer/libaur/printer.py", line 176, in pretty_print_info
    tw = _get_term_width()
  File "/tmp/pacaurtmp-remy/python-pywer-git/src/pywer/libaur/printer.py", line 52, in _get_term_width
    a, termwidth = stty.read().split()
ValueError: need more than 0 values to unpack

----------------------------------------------------------------------
Ran 5 tests in 0.015s

FAILED (errors=1)
==> ERROR: A failure occurred in check().
    Aborting...
:: python-pywer-git cleaned
```

**Comment by rmarquis** [comment]
Using the [undocumented ```--ask``` pacman option](https://lists.archlinux.org/pipermail/pacman-dev/2015-April/020057.html) might actually be the exact needed feature to solve this issue.

**Comment by rmarquis** [comment]
Using the ```PACMAN``` env variable might be a workaround, although it might have break support for self defined pacman binaries (chaining some other third-party tools like powerpill).

**Comment by rmarquis** [comment]
Done in db50bc23a92df374934226b48953c4e196f0ee9c. Since the ```PACMAN``` variable isn't exported, external utilities should still behave properly.

**Comment by rmarquis** [comment]
Reopening. Conflicts aren't handled correctly (see expac and expac-git).

**Comment by rmarquis** [comment]
This has been reverted in 57a2eaa731fd57d6bfa7f488ad449c64cd241447 for now.

**Comment by rmarquis** [comment]
Fixed in 12707cc7f9fb733082dcb33e22e4994c11eabb5f.
